### DISCUSSION TITLE
Atom Notes in MinimalLib possible?

### DISCUSSION THREAD

**Discussion by heberleh** [rdkit/rdkit#5187] [Q&A] [answered]
I saw that an `atomLabels` vector can be set in the `drawingOpts` in the Javascript version, but `atomNote` is set on an atom basis in the original RDKit (not as a vector in the DrawingOpts).

Any current way to set atom notes?
![image](https://user-images.githubusercontent.com/4249107/162784803-83f5e1c3-d886-46e2-9239-156e5fa0b672.png)

**Comment by ptosco** [comment, marked as answer]
@heberleh You can use CXSMILES:
```
rdkitModule.get_mol('O=C([O-])c1cncnc1 |atomProp:0.atomNote.-0.24:1.atomNote.0.11:2.atomNote.0.04:3.atomNote.0.04:4.atomNote.0.03:5.atomNote.-0.54:6.atomNote.0.07:7.atomNote.-0.54:8.atomNote.-0.24|')
```
